.github/workflows/lint.yml

```
@@ -0,0 +1,169 @@
+# ============================================================================ #
+#
+# Run `clang-tidy' over the codebase and report warnings as annotations.
+#
+# ---------------------------------------------------------------------------- #
+
+name: 'Lint'
```

```suggestion
name: 'Pkgdb Lint'
```
This isn't running the Rust linter, so we should be explicit about that.